### PR TITLE
Add receipt data fields for TRA-06 compliance

### PR DESCRIPTION
**Pull request by adriantimberlid**

### CODE REVIEW

File: openapi/integrator/merchant/partner.yaml [hunk 1]
```diff
@@ -213,6 +215,18 @@ components:
...
+        responseCode:
+          type: string
+          maxLength: 2
+          description: >-
+            Response code from issuer or processor when refund status is Approved or Declined
+        authorisationCode:
+          type: string
+          maxLength: 6
+          description: >-
+            Authorisation code: When the refund is authorised successfully, this field holds the authorisation code for the refund. When the refund is not authorised, this field is empty.
```
Comment (thxmasj) [inline openapi/integrator/merchant/partner.yaml:229]: And similar for this one. Common description could be: `A code from the issuer identifying a successful authorisation`.

File: openapi/integrator/merchant/partner.yaml [hunk 2]
```diff
@@ -273,3 +287,9 @@ components:
         - Approved
         - Declined
         - Failed
+
+    stan:
+      type: string
+      maxLength: 6
+      description: >-
+        System Trace Audit Number (STAN): This field is a number assigned by the message originator to identify the transaction.
```
Comment (thxmasj) [inline openapi/integrator/merchant/partner.yaml:295]: `System trace audit number; a number assigned by the payment initiator to identify the transaction.`

**Review by thxmasj — CHANGES_REQUESTED** [review]
Suggested alternative descriptions for the fields so they can be reused, otherwise LGTM.

File: openapi/integrator/merchant/partner.yaml (hunk 2 above)
Comment (adriantimberlid) [reply openapi/integrator/merchant/partner.yaml:295]: Sure. I found this in the NISOK description of the field.

File: openapi/integrator/merchant/partner.yaml [hunk 3]
```diff
@@ -213,6 +215,18 @@ components:
           format: date-time
           description: >-
             The time and date the request was created. The date fields use the YYYY-MM-DDThh:mm:ss.sssZ date ISO 8601 format. Times are returned in the UTC time zone.
+        stan:
+          $ref: '#/components/schemas/stan'
+        responseCode:
+          type: string
+          maxLength: 2
+          description: >-
+            Response code from issuer or processor when refund status is Approved or Declined
```
[the remark this answers is not in the document]
Comment (adriantimberlid) [reply openapi/integrator/merchant/partner.yaml:224]: That is true. Reusing if fine by me and I will make this change.

**Comment by adriantimberlid** [comment]
> Suggested alternative descriptions for the fields so they can be reused, otherwise LGTM.

Changed to your suggestions, thanks!

**Review by thxmasj — APPROVED** [review]

**Pull request merged**